pkg/virt-handler/vm_test.go

```diff
@@ -3365,3 +3498,42 @@ func addNode(client *fake.Clientset, node *k8sv1.Node) {
 		return true, node, nil
 	})
 }
+
+func getDefaultMigrationConfiguration() *v1.MigrationConfiguration {
+	parallelOutboundMigrationsPerNode := virtconfig.ParallelOutboundMigrationsPerNodeDefault
+	parallelMigrationsPerCluster := virtconfig.ParallelMigrationsPerClusterDefault
+	allowAutoConverge := virtconfig.MigrationAllowAutoConverge
+	bandwidthPerMigration := resource.MustParse(virtconfig.BandwithPerMigrationDefault)
+	completionTimeoutPerGiB := virtconfig.MigrationCompletionTimeoutPerGiB
+	progressTimeout := virtconfig.MigrationProgressTimeout
+	unsafeMigrationOverride := virtconfig.DefaultUnsafeMigrationOverride
+	allowPostCopy := virtconfig.MigrationAllowPostCopy
+
+	return &v1.MigrationConfiguration{
+		ParallelOutboundMigrationsPerNode: &parallelOutboundMigrationsPerNode,
+		ParallelMigrationsPerCluster:      &parallelMigrationsPerCluster,
+		AllowAutoConverge:                 &allowAutoConverge,
+		BandwidthPerMigration:             &bandwidthPerMigration,
+		CompletionTimeoutPerGiB:           &completionTimeoutPerGiB,
+		ProgressTimeout:                   &progressTimeout,
+		UnsafeMigrationOverride:           &unsafeMigrationOverride,
```

Comment:
We should not allow a user to opt out here. Let's be secure by default, unless we see a need for this.

Reply:
We are secure by default, since `UnsafeMigrationOverride` defaults to false :)
Please note that this option already exists on a global level, this PR just makes it configurable at the policy level.